libs/qscintilla/Qt4Qt5/qscintilla.pro

```diff
@@ -27,6 +27,13 @@ TARGET = qscintilla2
 CONFIG += qt warn_off thread exceptions hide_symbols staticlib debug_and_release
 INCLUDEPATH += . ../include ../lexlib ../src
 
+contains(QT_VERSION, ^4\\..*\\..*) {
+        QMAKE_CXXFLAGS += -std=c++0x
+}
+contains(QT_VERSION, ^5\\..*\\..*) {
+        QMAKE_CXXFLAGS += -std=c++11
+}
```

Comment:
This should be simplified to just the following because we don't support Qt 4.x anyway.
```
QMAKE_CXXFLAGS += -std=c++11
```